configurator/configurator-api/src/main/java/org/codice/ddf/admin/configurator/Result.java

```diff
@@ -41,12 +41,12 @@
      *
      * @return Optional failure exception
      */
-    Optional<Throwable> getBadOutcome();
+    Optional<Throwable> getError();
```

Comment:
No good reason at all. I was trying to be overly generic when in fact, it makes more sense to make it a `ConfiguratorException`. Will change.